### PR TITLE
export trace id constant in api package

### PR DESCRIPTION
**Pull request by wperron**
**What this PR does**:

Export the `URLParamTraceID` constant in the `api` package as this constant was already exported in the `util` package prior to version 1.2.

**Checklist**
- ~[ ] Tests updated~
- ~[ ] Documentation added~
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by joe-elliott** [comment]
I'm good on this. You just need to sign the CLA and we'll merge.

**Comment by wperron** [comment]
> I'm good on this. You just need to sign the CLA and we'll merge.

Yup, just waiting on internal approval on that front, should be done later this week 👍

**Review by kvrhdn — APPROVED** [review]

**Pull request merged**
